File: atividades em python/exercícios 2° ano/class.py
class eletro:
    def __init__(self, ligado, voltagem, consumo):
        self.ligado=ligado
        self.voltagem=voltagem 
        self.consumo=consumo
    def isligado(self):
        return (self.ligado)
    def setligado(self, ligado):
        self.ligado=ligado

PS2= eletro(True,60,120)
TV=eletro(False, 100, 220)

print("O Play 2 está ligado?", PS2.isligado())
print(PS2.consumo)
print("A TV está ligada?", TV.isligado())


# self.variavel E um atributo, variavel E um valor

File: atividadesemjs/class.js
class Eletro {
    constructor(ligado, voltagem, consumo) {
        this.ligado = ligado;
        this.voltagem = voltagem;
        this.consumo = consumo;
    }
    isLigado() {
        return this.ligado;
    }
    setLigado(ligado) {
        this.ligado = ligado;
    }
}

const PS2 = new Eletro(true, 60, 120);
const TV = new Eletro(false, 100, 220);

console.log("O Play 2 está ligado?", PS2.isLigado());
console.log(PS2.consumo);
console.log("A TV está ligada?", TV.isLigado());
console.log(TV.voltagem);
console.log("Ligando a TV...");
console.log(TV.setLigado(true));
console.log("A TV está ligada?", TV.isLigado());
